wolfssl/wolfcrypt/asn_public.h

```diff
@@ -229,6 +229,7 @@ typedef struct Cert {
     int      sigType;                   /* signature algo type */
     CertName issuer;                    /* issuer info */
     int      daysValid;                 /* validity days */
+    int      yearsValid;                /* validity years */
```

Comment:
John, since this is a public structure it might make sense to put this at the bottom for compatibility reasons?

Reply:
Good point.